mautic/mautic
# TPROD-299 Replace old TranslatorInterface

The TranslatorInterface has been removed in favor of Symfony\Contracts\Translation\TranslatorInterface



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ x ]
| Automated tests included?              | [ ] 



#### Description:

https://mautic.atlassian.net/browse/TPROD-299

The TranslatorInterface has been removed in favor of Symfony\Contracts\Translation\TranslatorInterface
There's a couple of BC breaks:

- All userland code that overrides part of the Mautic will need to change to new class. 
- All translatable items must be a string



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check all text is translated.
